docs/protocols/java_object_graph_spec.md

```diff
@@ -69,19 +84,49 @@ If schema evolution mode is enabled globally or enabled for current class, class
 - If meta share mode is enabled, class will be written as a unsigned int.
```

Comment:
The sentences feel a little confusing. Perhaps we can change it to:
```
If meta share mode is not enabled, class meta will be written as scheme consistent mode. Additionally， field meta such as field type
  and name will be written when the object value is being serialized using a key-value like layout.
```

Reply:
updated